src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -2092,6 +2092,11 @@ export class SearchView extends ViewPane {
 	}
 
 	private _saveSearchHistoryService() {
+		// {{SQL CARBON EDIT}}
```

Comment:
Please add a comment with a description and link to the issue where this was fixed in VS Code so we know to remove the edit tag once that commit is brought in.